packages/calypso-products/src/plans-list.tsx

```diff
@@ -723,8 +724,11 @@ const getPlanPersonalDetails = (): IncompleteWPcomPlan => ( {
 	group: GROUP_WPCOM,
 	type: TYPE_PERSONAL,
 	getTitle: () =>
-		// translators: Starter is a plan name
-		i18n.translate( 'Starter' ),
+		getPlansListExperiment( 'wpcom_plan_name_change_starter_to_beginner' ) === 'treatment'
+			? // translators: Starter is a plan name
+			  i18n.translate( 'Beginner' )
```

Comment:
Should probably use `Beginner` for the translators not.

Reply:
[Fixed](https://github.com/Automattic/wp-calypso/pull/89535/commits/24dced77a97ddead542aa2ac9132daf4dd4f6fe5)